docs/tools/new_tools_temp.md

```diff
@@ -46,23 +46,49 @@ Use pip to install:
     python -m pip install mbed-tools --pre
     ```
 
+## Set environment variables
+
+<!--like what? when? why? should this be with installation instructions?-->
+
+To override default environment variables:
+
+```
+mbed-tools env <VARIABLE>=<value>
```

Comment:
What's this? `mbedtools env` just prints information about the environment variables you can set if you want to access private boards in the online database, it doesn't allow you to set variables like that...

Reply:
So. How and why do you actually use this idea? Because the impression I got from the original text is that env is how you modify things, not list them.

Reply:
You can either specify the variables in a `.env` file and the tool will discover them, or you can set them per terminal session the same way you would set any environment variable https://www.schrodinger.com/kb/1842. The "why" is what `mbedtools env` tells you, as pasted in the comment below. `mbedtools env` doesn't list any set variables, it only describes the variables the tool recognises and why you might need to set them (most users will not need to set them at all).

Reply:
So the command is `setenv`, rather than `env`? (that's according to the KB you linked to)